libs/librepcb/libraryeditor/sym/dialogs/symbolpinpropertiesdialog.cpp

```diff
@@ -55,6 +55,9 @@ SymbolPinPropertiesDialog::SymbolPinPropertiesDialog(SymbolPin& pin,
   mUi->spbPosY->setValue(mSymbolPin.getPosition().getY().toMm());
   mUi->spbRotation->setValue(mSymbolPin.getRotation().toDeg());
   mUi->spbLength->setValue(mSymbolPin.getLength()->toMm());
+      
+  // preselect name
+  mUi->edtName->setSelection(0, mUi->edtName->text().size());
```

Comment:
Is there a reason for not using [`selectAll()`](https://doc.qt.io/qt-5/qlineedit.html#selectAll)? Would be cleaner and simpler IMHO...

Reply:
I don't know the function. Of course it's nicer :-) Thanks for the hint.